### PR TITLE
feat: Add AppierAdminAPI

### PR DESCRIPTION
**Pull request by joaofsl**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-util-vue/issues/40 |

### CODE REVIEW

File: js/model.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        }
+
+        async deleteModel(name) {
+            const url = `${this.baseUrl}admin/models/${model}/${name}`;
+            const response = await this.delete(url);
+            return response;
+        }
+    };
+
+export default ModelAPI;
```
Comment (gcandal) [inline js/model.js]: Run `cleanup . ` and/or `yarn pretty`

**Comment by gcandal** [comment]
@joamag

**Comment by joamag** [comment]
@gcandal this should be changed to reflect https://github.com/hivesolutions/appier_admin conf variables and structure

**Comment by gcandal** [comment]
@joamag this mimics https://github.com/hivesolutions/appier_extras_api/blob/master/src/appier_admin/base.py . It has the same name. If you want we can move this to a `appier_extras_api_js`

**Review by joamag — APPROVED** [review]

**Comment by joamag** [comment]
@gcandal @joaofsl renamed the project and changed some class names

**Pull request merged**